decidim-core/app/views/layouts/decidim/_social_meta.html.erb

```diff
@@ -0,0 +1,11 @@
+<meta name="twitter:card" content="summary_large_image" />
+<meta name="twitter:site" content="@<%= content_for(:twitter_handler) %>" />
+<meta name="twitter:title" content="<%= content_for(:proposal_title) %>" />
```

Comment:
I totally agree, we should allow any view to declare what it's social title is. More than that: I think we could even unify `content_for(:social_title)` to just `content_for(:title)` and use the same info on the page's `<title>` tag (it would actually happen just like that if we omitted the `meta` tag entirely, but it's cleaner this way).